src/styles.css

```diff
@@ -319,6 +371,10 @@ button {
   height: 400px;
 }
 
+.ingredient-list__item {
+  margin: 1em 0;
```

Comment:
I guess devs typically choose to work with either px or em (or rem) - not sure if this is something we care about at this time but something to keep in mind for future projects :)

Reply:
Good point, I will change those to px. I chose em because that was the default value in the browser that the reset file got rid of, but I can convert it to pixels.